tests/unit/enums/test_standard_sql_data_types.py

```diff
@@ -59,15 +59,3 @@ def test_standard_sql_types_enum_members(enum_under_test, gapic_enum):
     for name in ("STRUCT", "ARRAY"):
         assert name in gapic_enum.__members__
         assert name not in enum_under_test.__members__
-
-
-def test_standard_sql_types_enum_docstring(enum_under_test, gapic_enum):
-    assert "STRUCT (int):" not in enum_under_test.__doc__
-    assert "BOOL (int):" in enum_under_test.__doc__
-    assert "TIME (int):" in enum_under_test.__doc__
-
-    # All lines in the docstring should actually come from the original docstring,
-    # except for the header.
-    assert "An Enum of scalar SQL types." in enum_under_test.__doc__
-    doc_lines = enum_under_test.__doc__.splitlines()
-    assert set(doc_lines[1:]) <= set(gapic_enum.__doc__.splitlines())
```

Comment:
This was removed, because the new generated enum has no docstring, at least not at the time of writing.

Reply:
Bummer. We'll really want those at some point. Filed https://github.com/googleapis/gapic-generator-python/issues/625 against the microgenerator.

Reply:
Make sense. I will add ti back, but mark it as skipped. It will then be easy to resurrect it when the generator is fixed.